js/main.js

```diff
@@ -23,31 +23,52 @@ maps.forEach(map_ => {
         filter: ['grayscale:100%'],
     }).addTo(name);
 
+
     if (map_ === "map0") {
         //crop field map
 
+       
         let farm_field_data  = {
             "Tobacco": { 
                data: tobacco_fields
             },
             "Soybean": {
-                data: soybeans_feilds
+                data: soybeans_feilds, 
             },
             "Maize": {
-                data: maize_feilds
-            }
+                data: maize_feilds, 
+            },
+            "EPAs": {
+                data: EPAs_data,
+            },
+            "Districts": {
+                data: districts_data,
+            },
+
         }
 
         let baseOne = {}
 
         Object.keys(farm_field_data).forEach(name_ => {
-            let fieldData = L.geoJson(farm_field_data[name_]["data"], {
+            let fieldData = L.geoJson(farm_field_data[name_]["data"],{ 
```

Comment:
The outline was requested for districts and EPAs at @Ugerpaul17 . Can we have only those having the red. Also, it should also be only the outline that is red and not the fill color as well. Kindly make sure that is the case

Reply:
the Project manager said I should make the field color yellow and the outline red for districts and Epas with a transparent fill color. maybe you can get back to her and ask her why...
![Screenshot (78)](https://user-images.githubusercontent.com/25029435/85672311-6f4aa280-b6cb-11ea-9430-307ac7d6a548.png)
![Screenshot (79)](https://user-images.githubusercontent.com/25029435/85672366-7c679180-b6cb-11ea-8ea9-f6965f662920.png)

Reply:
Why would I ask the PM about these changes?